### PR TITLE
Cleanup frontend code

### PR DESCRIPTION
**Pull request by klochowicz**
Remove the `APP` variable from `maker-frontend` (from now on, it only builds maker).
Update README, CI and start_all.sh script.

note: the old taker component will be remove in the next PR, as I'm still mining it for ideas.

### CODE REVIEW

File: .github/workflows/build-release-binary.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ jobs:
         run: |
           cd maker-frontend;
           yarn
-          APP=maker yarn build
+          yarn build
```
Comment (thomaseizinger) [inline .github/workflows/build-release-binary.yml]: We could write this as:

```suggestion
          cd ${{ matrix.bin }}-frontend;
          yarn
          yarn build
```

And get rid of the `if`.

File: maker-frontend/vite.config.ts [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
-if (!app || (app !== "maker" && app !== "taker")) {
-    throw new Error("APP environment variable needs to be set to `maker` `taker`");
-}
-
-const backendPorts = {
-    "taker": 8000,
-    "maker": 8001,
-};
+const app = "maker";
+const backendPort = 8001;
```
Comment (thomaseizinger) [inline maker-frontend/vite.config.ts]: I guess this indirection can also go away now!

**Review by thomaseizinger — APPROVED** [review]

File: .github/workflows/build-release-binary.yml (hunk 1 above)
Comment (klochowicz) [reply .github/workflows/build-release-binary.yml]: wouldn't this also make it a bit harder to find out which job failed? (right now they're named separately, so you see at the first glance which frontend failed on the CI)

edit: nevermind, I see the point in avoiding needless duplication too ;) Added as a separate commit.

**Pull request merged**